minard/cssProc.py

```diff
@@ -0,0 +1,117 @@
+import contextlib
+import shutil
+import tempfile
+import argparse
+import sys
+import os
+import os.path
+import time
+import re
+import string
+import subprocess
+import datetime
+import json
+
+import psycopg2
+import psycopg2.extras
+import psycopg2.extensions
+
+from .db import engine_nl
+from .views import app
+
+
+RUN_TYPES = {
+    0:"Maintenance",
+    1:"Transition",
+    2:"Physics",
+    3:"Deployed Source",
+    4:"External Source",
+    5:"ECA",
+    6:"Diagnostic",
+    7:"Experimental",
+    8:"Supernova",
+    }
```

Comment:
I think a copy of these is already stored in https://github.com/snoplus/minard/blob/master/minard/nlrat.py so you could just import it from there.